scripts/benchmark.py

```diff
@@ -36,11 +36,16 @@ def load_json(filename):
 
     return data
 
+allowed_geometries = ['filled_box', 'hollow_box']
```

Comment:
`geometries.keys()`

Reply:
Don't understand.

Reply:
OK, understood now. `requirements.txt` only to contain non-core libraries. Surprisingly to me, `json` is part of core.